### PR TITLE
Update Color class ref documentation

### PR DESCRIPTION
**Pull request by mrawlingst**
Improved the documentation for Color of the class reference documentation.
Added more clarifications to functions such as float or which format the color profile would be using/giving.

### CODE REVIEW

**Comment by toger5** [comment]
The changes are great!
Thank you. Would be great if you squash it to one commit.

like so
```
git reset HEAD~4
git commit -a -m "Update Color class ref documentation"
```

**Comment by mrawlingst** [comment]
I'm still adding more stuff based on feedbacks in discord but I'll squash once we've agreed that Color class is considered 'completed'.

File: doc/base/classes.xml [hunk 1]
```diff
@@ -12217,7 +12217,11 @@
 			<argument index="3" name="a" type="float">
 			</argument>
 			<description>
-				Construct the color from an RGBA profile.
+				Constructs a color from an RGBA profile using values between 0 and 1 (float).
+
```
Comment (akien-mga) [inline doc/base/classes.xml]: Could you amend the commit to remove the empty lines before code blocks? Those will be parsed as an empty paragraph in the docs, adding useless spacing.

**Comment by mrawlingst** [comment]
Updated! Thanks for the info. I'll keep that in mind for the next documentation.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -12277,10 +12277,10 @@
 </class>
 <class name="Color" category="Built-In Types">
 	<brief_description>
-		Color in RGBA format.
+		Color in RGBA format with some support for ARGB format.
```
Comment (akien-mga) [inline doc/base/classes.xml]: I think it's bogus that the API uses ARGB instead of RGBA, but it's indeed the case currently so the docs are correct :)

**Pull request merged**